### PR TITLE
GREEN-update-docs-with-prospect-pool-related-changes

### PR DESCRIPTION
**Pull request by davidhan527**
https://greenhouseio.atlassian.net/browse/GREEN-10399

### CODE REVIEW

File: source/includes/candidate-ingestion/_candidates.md [hunk 1]
```diff
@@ -170,17 +173,18 @@ referrer.email | String | Mixed | Used to match this referrer with an existing G
 referrer.first_name | String | Mixed | Used to create a new ‘Referrer’ in Greenhouse if referrer.email does not match an existing user. Only required if referrer is included.
 referrer.last_name  | String | Mixed | Used to create a new ‘Referrer’ in Greenhouse if referrer.email does not match an existing user. Only required if referrer is included.
 notes | String | No | Free-form plain-text notes about this candidate. One way for this to be used is to send secondary information that our API can’t capture as structured data. For example: "Skills: Java, C++, Python"
+prospect_pool_id | Integer | Mixed | Used to put a prospect in the Prospect Pool with this id. Only required if prospect_pool_stage_id is included. prospect property must be set to true.
```
Comment (MarkyMarkMcDonald) [inline source/includes/candidate-ingestion/_candidates.md]: Can we capitalize the start of the second sentence in these three properties?

File: source/includes/candidate-ingestion/_prospect_pools.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+                "name": "In Discussion",
+            }
+        ]
+    }
+]
+```
+
+Retrieve prospect pools along with their stages for the current user's organization.
+
+`GET https://api.greenhouse.io/v1/partner/prospect_pools`
```
Comment (MarkyMarkMcDonald) [inline source/includes/candidate-ingestion/_prospect_pools.md]: Should we add a note that stages will be returned in the order they should be displayed to the user?

File: source/includes/candidate-ingestion/_candidates.md (hunk 1 above)
Comment (davidhan527) [reply source/includes/candidate-ingestion/_candidates.md]: I didn't capitalize it bc the property name in the docs is also not capitalized. Maybe i'll say "property"

**Comment by davidhan527** [comment]
Hey @tdphillipsjr , this PR is good to go. If all is good, please merge this PR.

File: source/includes/candidate-ingestion/_prospect_pools.md (hunk 2 above)
Comment (tdphillipsjr) [reply source/includes/candidate-ingestion/_prospect_pools.md]: @davidhan527 - Did you see this one?  Just clarifying you're not doing this before I merge.

Comment (davidhan527) [reply source/includes/candidate-ingestion/_prospect_pools.md]: Yeah I made that change in the description here - https://github.com/grnhse/greenhouse-api-docs/pull/151/commits/3c17aa0b231bcaa3a6aa2cda4bd89ec6f7fee807#diff-08b6358a993e825b70e3384c94d00bb9R177

Thanks for checking.

Comment (tdphillipsjr) [reply source/includes/candidate-ingestion/_prospect_pools.md]: 👍

**Pull request merged**